### PR TITLE
New module to setup cloud connector properties

### PR DESCRIPTION
**Pull request by grastogi23**
##### ISSUE TYPE

 - New Module Pull Request

##### COMPONENT NAME

avi.avi_cloudconnectoruser

##### ANSIBLE VERSION

```
ansible 2.4.0
  config file = /home/grastogi/.ansible.cfg
  configured module search path = [u'/home/grastogi/.ansible/plugins/modules', u'/usr/share/ansible/plugins/modules']
  ansible python module location = /home/grastogi/ansible2.3/local/lib/python2.7/site-packages/ansible
  executable location = /home/grastogi/ansible2.3/bin/ansible
  python version = 2.7.6 (default, Jun 22 2015, 17:58:13) [GCC 4.8.2]
```

##### SUMMARY

New module to setup cloud connector properties.

```

```

### CODE REVIEW

**Comment by khaltore** [comment]
shipit

**Comment by ericsysmin** [comment]
shipit

**Comment by ericsysmin** [comment]
shipit

**Comment by khaltore** [comment]
shipit

**Comment by gundalow** [comment]
Closing and reopening to retrigger CI as we've extended module validation rules.

File: lib/ansible/modules/network/avi/avi_cloudconnectoruser.py [hunk 1]
```diff
@@ -0,0 +1,121 @@
...
+except ImportError:
+    HAS_AVI = False
+
+
+def main():
+    argument_specs = dict(
+        state=dict(default='present',
+                   choices=['absent', 'present']),
+        name=dict(type='str', required=True),
+        private_key=dict(type='str',),
```
Comment (gundalow) [inline lib/ansible/modules/network/avi/avi_cloudconnectoruser.py:104]: I wonder if this (and any other instance of `private_key`) should have `no_log=true`

I might have missed this in some other PRs, so feel free to raise one PR that updates all the modules.

**Review by gundalow — CHANGES_REQUESTED** [review]

**Comment by grastogi23** [comment]
Sure, let me scrub all of them. I did take care of those in sslkeycertificate before.

**Comment by gundalow** [comment]
I believe that's all the Avi modules merged. You are now free to raise a single PR that updates all modules for any improvements, such as:
* Use of `no_log`correct private keys 
* Formatting in DOCUMENTION block

**Pull request merged**